frontend/src/model/overview/FormEditPage.tsx

```diff
@@ -74,6 +79,26 @@ export default function FormEditPage({ model }: FormEditPageProps) {
     setUnsavedChanges(isEdit)
   }, [isEdit, setUnsavedChanges])
 
+  function handleJsonFormOnSubmit(formData: string) {
+    setJsonUploadDialogOpen(false)
+    try {
+      const jsonInput = JSON.parse(formData)
+      if (schema) {
+        const steps = getStepsFromSchema(schema, {}, ['properties.contacts'], JSON.parse(jsonInput))
```

Comment:
`properties.contacts` does not exist in V2 I don't think?  This can just be an empty array.